telemetry/definitions/commonDefinitions.json

```diff
@@ -2609,50 +2589,74 @@
             ],
             "passive": true
         },
+
         {
-            "name": "awsq_serviceInvocation",
-            "description": "Client side invocation of the weaverbird service for solving software engineering tasks",
-            "metadata": [
-                {
-                    "type": "awsqAssignClickCount"
-                },
-                {
-                    "type": "awsqGenerateCodeIterationCount"
-                },
-                {
-                    "type": "awsqGenerateCodeClickCount"
-                },
-                {
-                    "type": "awsqApproachIterationCount"
-                },
-                {
-                    "type": "awsqIsRegeneratedCode"
-                },
-                {
-                    "type": "awsqFilesReviewedCount"
-                },
-                {
-                    "type": "awsqIsAcceptedCodeChanges"
-                },
-                {
-                    "type": "awsqRepositorySize"
-                },
-                {
-                    "type": "awsqFilesChangedCount"
-                },
-                {
-                    "type": "awsqThumbsUpCount"
-                },
-                {
-                    "type": "awsqThumbsDownCount"
-                },
-                {
-                    "type": "awsqIsApproachAccepted"
-                },
-                {
-                    "type": "awsqServiceErrorsCount"
-                }
-            ]
+            "name": "awsq_assignCommand",
+            "description": "User have triggered /assign",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqAssignCommandCount" }]
+        },
+        {
+            "name": "awsq_codeReGeneration",
+            "description": "User invoked the weaverbird agent to re-engage in Code gen",
+            "unit": "Count",
+            "metadata": [{ "type": "awsqCodeReGenerationCount" }]
```

Comment:
awsqAssignCommandCount is unnecessary because the metric is defined with unit:Count.

awsqCodeReGenerationCount is unnecessary because the metric is defined with unit:Count.

Reply:
removed